specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -201,7 +205,8 @@
             "$ref": "./examples/ContainerGroupsDelete.json"
           }
         },
-        "description": "Delete container groups.",
+        "summary": "Delete the specified container groups.",
+        "description": "Delete the specified container groups in the specified subscription and resource group. The operation does not delete other resources created as part of creating a container group such as volumes. All other resources created with the container group are part of the same resource group and can be deleted individually.",
```

Comment:
Is this added detail correct about not deleting the volumes associated and any IP addresses/etc. being in the same RG?

Reply:
Should be fine, as long as it's helpful for the customer.

Reply:
Volumes/image registry credentials are provided by the user when creating a container group. But IP is different. It's provided by ACI. Once a container group is deleted, the IP will be recycled. I think it would be more clear to say `The operation does not delete any resources provided by the users such as volumes`